includes/class-wporg-gp-translation-events-route.php

```diff
@@ -123,6 +126,22 @@ public function events_edit( $event_id ) {
 		$event_status                  = $event->post_status;
 		list( $permalink, $post_name ) = get_sample_permalink( $event_id );
 		$permalink                     = str_replace( '%pagename%', $post_name, $permalink );
+		$show_delete_button = false;
+		if ('Edit Event' === $event_form_title) {
+			$event = get_post( $event_id );
```

Comment:
We can remove this if condition since we are already in the `events_edit` function. Also, `$event` has been assigned a value above so it's okay to remove this duplicate.

Reply:
Done!